modules/react/button/stories/stories_ToolbarButtons.tsx

```diff
@@ -43,4 +44,50 @@ storiesOf('Components/Buttons/Toolbar', module)
         <div style={customElementStyles}>Normal</div>
       </ToolbarDropdownButton>
     </div>
+  ))
+  .add('Toolbar Dropdown Button with Menu', () => (
+    <div className="story">
+      <h3>Toolbar Dropdown Button with Menu</h3>
+      <Menu>
+        <Tooltip title="Expand">
+          <Menu.Target
+            as={ToolbarDropdownButton}
+            icon={zoominIcon}
+            onClick={() => {
+              console.log('Expand icon clicked');
+            }}
+          ></Menu.Target>
+        </Tooltip>
+        <Menu.Popper>
+          <Menu.Card>
+            <Menu.List>
+              <Menu.Item
+                aria-label="Expand All"
```

Comment:
aria label is not needed unless we have an icon.

Reply:
oh yes, makes sense! My bad there!